### PR TITLE
Make it possible to use without exceptions by defining CMRC_NO_EXCEPTIONS

### PR DESCRIPTION
**Pull request by jorgen**

### CODE REVIEW

**Comment by Pospelove** [comment]
Nice feature

File: CMakeRC.cmake [hunk 1]
```diff
@@ -77,6 +77,10 @@ set(hpp_content [==[
 #include <system_error>
 #include <type_traits>
 
+#if !(defined(__EXCEPTIONS) || defined(__cpp_exceptions) || defined(_CPPUNWIND))
```
Comment (vector-of-bool) [inline CMakeRC.cmake:80]:
```suggestion
#if !(defined(__EXCEPTIONS) || defined(__cpp_exceptions) || defined(_CPPUNWIND) || defined(CMRC_NO_EXCEPTIONS))
```

**Review by vector-of-bool — CHANGES_REQUESTED** [review]
Effective and simple. Only one minor tweak requested.

File: CMakeRC.cmake (hunk 1 above)
Comment (jorgen) [reply CMakeRC.cmake]: absolutely

**Review by vector-of-bool — APPROVED** [review]

**Pull request merged**